common/crypto/tlsgen/ca.go

```diff
@@ -40,7 +40,10 @@ type CA interface {
 	// with a given custom SAN.
 	// The certificate is signed by the CA.
 	// Returns nil, error in case of failure
-	NewServerCertKeyPair(host string) (*CertKeyPair, error)
+	NewServerCertKeyPair(hosts ...string) (*CertKeyPair, error)
```

Comment:
I ended up generating a cert with SANs at one point. (127.0.0.1, ::1, and localhost). I put this in a separate commit to keep it isolated from the "real" change and to let me make use of it on top. If I didn't end up using it, oh well.

Reply:
Looks like I used it on line 186 of consenter_test.go. I guess it's a habit for me to use IP addresses in most of the certificates because I run development servers on the loopback adapter to firewall notifications popping up on my workstation.

Since you're not terribly opposed to it, I'd rather leave it than remove it.